src/main/telemetry/msp_shared.h

```diff
@@ -20,8 +20,8 @@
 
 #pragma once
 
-#define MSP_TLM_INBUF_SIZE 128
-#define MSP_TLM_OUTBUF_SIZE 128
+#define MSP_TLM_INBUF_SIZE MSP_PORT_OUTBUF_SIZE
```

Comment:
Do we really need such a big buffer? It is 4112B. Does it make sense to use `MSP_PORT_INBUF_SIZE` instead?

Reply:
@daleckystepan Good point. If ```USE_FLASHFS``` this buffer is huge. ```MSP_PORT_INBUF_SIZE``` for the input buffer should be fine

Reply:
Should we make another define to avoid allocating another 4112B when USE_FLASHFS ? or happy with the following?:

```#define MSP_TLM_INBUF_SIZE MSP_PORT_INBUF_SIZE 
#define MSP_TLM_OUTBUF_SIZE MSP_PORT_OUTBUF_SIZE```

Reply:
We could have a ```#define MSP_PORT_OUTBUF_SIZE_MIN 320``` and use that instead.

Reply:
happy with this? 
```
#define MSP_PORT_INBUF_SIZE 192
#define MSP_PORT_OUTBUF_SIZE_MIN 320

#ifdef USE_FLASHFS
#ifdef STM32F1
#define MSP_PORT_DATAFLASH_BUFFER_SIZE 1024
#else
#define MSP_PORT_DATAFLASH_BUFFER_SIZE 4096
#endif
#define MSP_PORT_DATAFLASH_INFO_SIZE 16
#define MSP_PORT_OUTBUF_SIZE (MSP_PORT_DATAFLASH_BUFFER_SIZE + MSP_PORT_DATAFLASH_INFO_SIZE)
#else
#define MSP_PORT_OUTBUF_SIZE MSP_PORT_OUTBUF_SIZE_MIN // As of 2021/08/10 MSP_BOXNAMES generates a 307 byte response for page 1.
#endif
```

Reply:
Yes 👍

Reply:
Done